_news/20230920-tstrait-0.0.1.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: news
+title: New software for simulating quantitative traits
+description: | 
+  Often we want to map a genotype to a phenotype. A new `tskit`-based
+  simulator leverages the succinct tree sequence format to do so
+  efficiently for large numbers of genomes.
+  
+  The new software, tstrait, takes an existing tree sequence containing
+  mutations, and simulates a quantitative phenotypic trait whose value
+  is partially determined by a selected set of causal mutations in the
+  tree sequence.
+
+  This is an initial 0.0.1 release.
+image: ../assets/images/news/20230905-tstrait/daiki.jpg
+timestamp: 2023-09-20
+author: Yan Wong
+---
+
+# Ts-trait: new software for simulating quantitative traits (phenotypes)
+
+![Daiki Tagami]({{ '/assets/images/news/20230920-tstrait-0.0.1/daiki.jpg' |  relative_url}}){:class="news-float-left"}
+
+Daiki Tagami, a student in the Kelleher group, has built [tstrait](https://tskit.dev/software/tstrait.html),
+a software package to layer simulated quantitative phenotype traits on top of an existing tree sequence. This
+should be useful, for example, to create large simulated datasets for testing genome-wide association (GWAS)
+methods.
+
+## Example code
+
```

Comment:
Add `{:style="break: both"}` as the line immediately after the `## Example code` line.